class UserNotFoundError(Exception):
    pass

class PostNotFoundError(Exception):
    pass